tests/integration/inhibit-if-kmods-is-not-supported/main.fmf

```diff
@@ -1,5 +1,5 @@
 summary: kmods tests
 # TODO Add prepare with running ansible playbooks
-tag+: [centos6, centos7, centos8]
+tag: [centos6, centos7, centos8, inhibit-if-kmods-is-not-supported]
```

Comment:
Hint: Instead of using `tag` for selecting relevant tests you might want to use the context [adjust](https://tmt.readthedocs.io/en/latest/spec/core.html#adjust) feature which gives a bit more flexibility:

```yaml
adjust:
    enabled: false
    when: distro == centos-8
```
Providing context on the command line works like this:

```
tmt --context distro=centos-8 run ...
```